core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/models/PurchasingData.android.kt

```diff
@@ -1,5 +1,12 @@
 package com.revenuecat.purchases.kmp.models
 
-import com.revenuecat.purchases.models.PurchasingData as RcPurchasingData
+import com.revenuecat.purchases.kmp.ProductType
+import com.revenuecat.purchases.kmp.toProductType
+import com.revenuecat.purchases.models.PurchasingData as AndroidPurchasingData
 
-public actual typealias PurchasingData = RcPurchasingData
+internal class AndroidPurchasingData(
+    wrapped: AndroidPurchasingData
```

Comment:
Both the import alias and our class are called `AndroidPurchasingData`. Maybe the import alias can be `NativeAndroidPurchasingData` or something, to avoid confusion?

Reply:
Also the constructor should be `internal`, otherwise we're exposing the native Android type.